cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -932,23 +930,30 @@ <h2 class="h3 u-mb15">
             {% elif card.advance_apr_median is not none
                 and card.advance_apr_min != card.advance_apr_max %}
                 <p>
-                    You’ll only know your exact cash advance APR if you are
-                    approved for the card. The median cash advance APR for
-                    this card is {{ apr(card.advance_apr_median) }}.
+                    The median cash advance APR for this card is
+                    {{ apr(card.advance_apr_median) }}.
+                </p>
+            {% endif %}
+            {% if card.cash_advance_fee_for_each_transaction %}
+                <p>
+                    The cash advance fee is charged for each time you get a cash
```

Comment:
Should this be "is charged each time" or "is charged for each cash advance"?

Reply:
Good catch, fixed in 4db5507